storage/pushsync/pushsync_simulation_test.go

```diff
@@ -59,14 +60,10 @@ func TestPushSyncSimulation(t *testing.T) {
 	nodeCnt := 4
```

Comment:
*Note* there is still some issue with the pushsync simulation tests being very slow. Especially for a low(!) number of nodes.

Reply:
I have a hard time understanding what we are simulating with 4 nodes... most probably depth is 0 or 1 with 4 nodes, so I don't think this simulation makes sense with less than 16 nodes. What do you think?

Reply:
I would say you are right if it was not the case that here small network showed the problem and likely still have issues, so we need both